tensorflow/lite/micro/kernels/unidirectional_sequence_lstm_test.cc

```diff
@@ -26,10 +26,6 @@ limitations under the License.
 namespace tflite {
 namespace testing {
 namespace {
-
-constexpr int kLstmMaxNumInputOutputTensors = 24 + 1;
```

Comment:
Can we put this constant back? Its useful at the call sites.

Reply:
Of course!